src/main/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImpl.java

```diff
@@ -29,7 +28,6 @@ public class FinancialTransactionServiceImpl implements FinancialTransactionServ
 
     private final FinancialTransactionRepository financialTransactionRepository;
```

Comment:
I see no blank line separating the code lines of the individual fields in the class. I'm sorry, I can't accept this.